queries.sql

```diff
@@ -143,12 +143,22 @@ SELECT vets.name, species.name FROM vets LEFT JOIN specializations ON vets.id =
 
 SELECT animals.name FROM animals JOIN visits ON animals.id = visits.animal_id WHERE visits.vets_id = 3 AND visits.date_of_visits BETWEEN '2020-04-01' AND '2020-08-30';
 
-select animal_id, count(*) from visits group by animal_id order by count(animal_id) desc limit 1;
+SELECT animal_id, count(*) from visits group by animal_id order by count(animal_id) desc limit 1;
 
 SELECT vets.name FROM vets JOIN visits ON vets.id = visits.vets_id WHERE visits.animal_id = 2 ORDER BY visits.date_of_visits LIMIT 1;
 
 SELECT animals.name, vets.name, visits.date_of_visits FROM animals JOIN visits ON animals.id = visits.animal_id JOIN vets ON visits.vets_id = vets.id ORDER BY visits.date_of_visits DESC LIMIT 1;
 
 SELECT COUNT(visits.animal_id) FROM visits JOIN animals ON visits.animal_id = animals.id JOIN vets ON visits.vets_id = vets.id LEFT JOIN specializations ON vets.id = specializations.vets_id LEFT JOIN species ON specializations.species_id = species.id WHERE species.id != animals.species_id;
 
-select species_id from animals where id = (select animal_id from visits where vets_id = 2 group by animal_id order by count(animal_id) desc limit 1); 
+SELECT species_id from animals where id = (select animal_id from visits where vets_id = 2 group by animal_id order by count(animal_id) desc limit 1); 
+
+
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vets_id = 2;
+SELECT * FROM owners where email = 'owner_ 18327@mail.com';
+
+EXPLAIN ANALYZE SELECT * FROM owners;
+EXPLAIN ANALYZE SELECT * FROM visits;
```

Comment:
- [ ] [REQUIRED] Good job so far! On your `PR` summary has `2` analyze results but it should be `3` because `EXPLAIN ANALYZE` command for each of the above queries (Screenshots should contain a total of 6 queries, you can combine queries in fewer screenshots):

The `first` screenshot should show how much time the query takes `BEFORE` the improvements introduced by student.
- It the query planner output you should see the usage of Seq scan and/or Filter
- The execution time should be around 1 sec (1000 ms).
The `second` screenshot should show how much time the query takes `AFTER` the improvements introduced by student.
- The execution time should be significantly lower than 1 sec (1000 ms).
- Please fix it 🙏🏼 

- `[HINT]`
```sql
EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
```

![image](https://user-images.githubusercontent.com/63915024/214321180-bc80621e-6d0c-4e0e-b90b-71b40033b2f2.png)

Reply:
Thanks, but in the screenshot you sent as an hint, the time is greater than 1000 for the second one